projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -53,6 +58,39 @@ export const settings = {
 				editor_type: isInsideSiteEditor ? 'site' : 'post',
 			} );
 
+		function getModalContent() {
+			const modalContent = {
+				title: __( 'Great progress!', 'jetpack' ),
+				body: __(
+					'You are one step away from bringing your site to life. Check out the next steps that will help you to launch your site.',
+					'jetpack'
+				),
+				actionButtonHref: launchPadUrl,
+				actionButtonTracksEvent: 'jetpack_launchpad_save_modal_next_steps',
+				actionButtonText: __( 'Next Steps', 'jetpack' ),
+			};
+
+			if ( siteIntentOption === 'newsletter' ) {
+				if ( isInsidePostEditor ) {
```

Comment:
We need to check the postType in the post editor here. Right now our check for the post editor above is simply `const isInsidePostEditor = document.querySelector( '.block-editor' ) !== null;`, but this would return `true` for the page editor as well.  We do not want this copy in the page editor, we would want the same copy in the page editor as in the site editor.

Reply:
I updated the PR to check for the `post` postType specifically